frontend/src/scenes/dashboard/Dashboards.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { dashboardsModel } from '~/models/dashboardsModel'
+import { Button, Spin } from 'antd'
+import { dashboardsLogic } from 'scenes/dashboard/dashboardsLogic'
+import { Link } from 'lib/components/Link'
+import { PlusOutlined } from '@ant-design/icons'
+
+export default function Dashboards() {
+    const { dashboardsLoading } = useValues(dashboardsModel)
+    const { dashboards } = useValues(dashboardsLogic)
+    const { addNewDashboard } = useActions(dashboardsLogic)
+
+    return (
+        <div>
+            <div style={{ marginBottom: 20 }}>
+                <Button onClick={addNewDashboard} style={{ float: 'right' }}>
+                    <PlusOutlined style={{ verticalAlign: 'baseline' }} />
+                    New Dashboard
+                </Button>
+                <h1>Dashboards</h1>
+            </div>
+
+            {dashboardsLoading ? (
+                <Spin />
+            ) : dashboards.length > 0 ? (
+                <table className="table">
```

Comment:
In line with our move to antd, I think worth moving this to an antd table. It'll also reduce LoCs and look better too